### PR TITLE
Default the To language dropdown to 'Select a language'

### PR DESCRIPTION
**Pull request by MaxSem**
Bug: https://phabricator.wikimedia.org/T214006

### CODE REVIEW

File: assets/translate.js [hunk 1]
```diff
@@ -1,3 +1,16 @@
+var alreadyUpdating = false,
+	enableInputs = function() {
+		alreadyUpdating = true;
```
Comment (mooeypoo) [inline assets/translate.js]: Is there a need for this variable? The loop below should be synchronous.

File: src/OOUI/TranslationsFieldset.php [hunk 2]
```diff
@@ -21,6 +23,7 @@ public function __construct(array $config = [])
         $currentFieldset = new FieldsetLayout();
         $fieldsets = [$currentFieldset];
         $prevTranslation = null;
+        $this->disabled = $config['disabled'] ?? false;
```
Comment (mooeypoo) [inline src/OOUI/TranslationsFieldset.php]: 'disabled' config is dealt with in Element.php, (FieldsetLayout -> Layout -> Element) so this shouldn't be needed.

File: assets/translate.js [hunk 3]
```diff
@@ -124,6 +142,11 @@ $( window ).on( 'load', function () {
 					}
 				} );
 			};
+
+		if ( targetLangCode !== 'fallback' ) {
+			inputWiget.setDisabled( false );
```
Comment (mooeypoo) [inline assets/translate.js]: Are we disabling inputs if the language is the fallback, or if the dropdown shows the "select language" ? it's not very clear from this condition, so I want to make sure.

Could an SVG not have a default language? (could it have only languages inside the switch?) if so, the inputs may be disabled accidentally. Do we want to make this condition be based on what the initial UI value is, instead? 

Either way, please add a quick comment in here; if I'm not 100% sure now, I imagine it won't be completely clear when we get back to it in a year :)

File: assets/translate.js (hunk 1 above)
Comment (MaxSem) [reply assets/translate.js]: Enabling a control counts as an update, resulting in preview being reloaded for every input being enabled.

File: src/OOUI/TranslationsFieldset.php (hunk 2 above)
Comment (MaxSem) [reply src/OOUI/TranslationsFieldset.php]: Doesn't seem to work for me.

File: assets/translate.js (hunk 1 above)
Comment (mooeypoo) [reply assets/translate.js]: So, re-reading the ticket and talking to Niharika, it sounds like the inputs should just always start out disabled -- which means we should probably just get them from the php-widgets already disabled, and enable them when the selection of the 'to' language changes.

File: src/OOUI/TranslationsFieldset.php (hunk 2 above)
Comment (mooeypoo) [reply src/OOUI/TranslationsFieldset.php]: Meh, that's not good for upstream, but I'll take a look later in OOUI while we use this here.

**Pull request merged**